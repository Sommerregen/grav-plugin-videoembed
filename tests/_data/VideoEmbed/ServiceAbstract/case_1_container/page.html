<h1>Test 1</h1>

<a href="[test_url]">[test_url]</a>