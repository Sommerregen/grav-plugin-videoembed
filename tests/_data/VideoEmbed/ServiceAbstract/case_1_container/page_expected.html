<h1>Test 1</h1>

<div class="container"><div>testOK</div></div>